src/app/components/post/post.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="singlePostWrapper">
+    <div class="image">Image</div>
+
+    <div class="info">
+        <h2>{{postTitle}}</h2>
+        <br>
+        <div class="postDateCreated">Posted  {{postDateCreated | dateAgo}} by <a class="postAuthor" href="">{{postAuthor}}</a> to <a class="postSubreddit" href="">{{postSubreddit | subredditFormat}}</a></div>
```

Comment:
This could use a little formatting. Did you try using prettier?

Reply:
I use just eslint and prettier eslint but it doesn't provide any fixes in html files. Can I fix it somehow?